Framework/include/Framework/Event.h

```diff
@@ -518,6 +523,7 @@ class Event {
    */
   ldmx::EventHeader eventHeader_;
 
+  const std::string eventHeaderPassName_{"eventPassName"};
```

Comment:
```suggestion
  const std::string eventHeaderPassName_{""};
```
I think we should keep the default to be an empty string

Reply:
Indeed, the pass name of the event header will be ignored since we never want to append a pass name to it.

https://github.com/LDMX-Software/ldmx-sw/blob/6aca7d804cc0f5fb555b4808fe10f01fc2dd41f7/Framework/include/Framework/Event.h#L181-L186

I would remove this member entirely and hardcode the empty string into the `nextEvent` source code.